docs/src/main/asciidoc/grpc-service-implementation.adoc

```diff
@@ -319,11 +319,48 @@ public class HelloServiceTest implements Greeter {
     Greeter client;
 
     @Test
-    void shouldReturnHello() throws Exception {
+    void shouldReturnHello() {
         CompletableFuture<String> message = new CompletableFuture<>();
         client.sayHello(HelloRequest.newBuilder().setName("Quarkus").build())
                 .subscribe().with(reply -> message.complete(reply.getMessage()));
         assertThat(message.get(5, TimeUnit.SECONDS)).isEqualTo("Hello Quarkus");
     }
 }
 ----
+
+== gRPC Server metrics
+
+=== Enabling metrics collection
+
+gRPC server metrics are automatically enabled when the application also uses the xref:micrometer.adoc[`quarkus-micrometer`] extension.
+It collects the metrics of all the gRPC services implemented by the application.
```

Comment:
When I read it, it's not clear to me what the `it` refers to. Is it the extension? (the same for the client, it's the same text there)